templates/cli/lib/commands/push.js.twig

```diff
@@ -1063,7 +1212,7 @@ const pushCollection = async ({ returnOnZero } = { returnOnZero: false }) => {
             attributes = await attributesToCreate(collection.remoteVersion.attributes, collection.attributes, collection);
 
             if (Array.isArray(attributes) && attributes.length <= 0) {
-                log(`No changes has been detected. Skipping ${collection.name} ( ${collection['$id']} )`);
+                log(`No destructive changes has been detected. Skipping ${collection.name} ( ${collection['$id']} )`);
```

Comment:
Do we have some non-destructive ones?

Reply:
If so, we shouldnt skip them. we should do them.
Also, in databases specifically, I would ALWAYS ask for confirmation, so they always see the table and can decide if all those changes should be applied - even if non-destructire. It's like having dry-run.

Reply:
The skipping is on the destructive changes, the non-destructive changes are being applied (as max)

Reply:
![image](https://github.com/appwrite/sdk-generator/assets/316103/6322fd97-2c21-4239-b4fd-0a61dd517850)